pkg/xds/sync/egress_proxy_builder.go

```diff
@@ -134,3 +132,23 @@ func (p *EgressProxyBuilder) Build(
 
 	return proxy, nil
 }
+
+func matchExternalPolicies(tags map[string]string, resources xds_context.Resources) (core_xds.PluginOriginatedPolicies, error) {
```

Comment:
nit: we probably should rename the method since it works for both internal and external services now

Reply:
renamed